debezium-embedded/src/test/java/io/debezium/embedded/EmbeddedEngineTest.java

```diff
@@ -149,6 +151,63 @@ public void shouldWorkToUseCustomChangeConsumer() throws Exception {
         stopConnector();
     }
 
+    @Test
+    public void shouldRunDebeziumEngine() throws Exception {
```

Comment:
I'd have expected existing tests to be adjusted to be coded against the new public interface?

Reply:
@gunnarmorling And this again was intentional - to verify that there is no impact on backward compatbility. The test rewrite should go hand in hand with removing deprecated interfaces.

Reply:
Ok, I see.